Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -56,9 +56,15 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
             var content = new Y.eZ.Content(),
                 version = new Y.eZ.Version(),
                 type = this.get('contentType'),
-                defaultFields = {};
+                defaultFields = {},
+                contentTypeNames,
+                contentTypeName;
 
-            content.set('name', 'New "' + this.get('contentType').get('names')['eng-GB'] + '"');
+            contentTypeNames = this.get('contentType').get('names');
+            contentTypeName = contentTypeNames[this.get('app').get('defaultLanguageCode')]
```

Comment:
Here logic is wrong.

It should iterate over prioritized languages until match is found, otherwise pick the one set as mainLanguage.
Just like in #550 

See also: https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/Helper/TranslationHelper.php#L164